cmd/agent/daemon/app/app.go

```diff
@@ -336,6 +339,18 @@ func (a *App) Run(ctx context.Context) error {
 		policy.Events = append(policy.Events, &ebpftracer.EventPolicy{
 			ID: events.NetFlowBase,
 		})
+
+		// If ebpf events exporters are not enabled but flows collection enabled
+		// we still need dns events to enrich dns question.
+		if len(exporters.Events) == 0 {
```

Comment:
This should be moved to a dependency of the `events.NetFlowBase` definition.

Reply:
Oh wait, we do deduplication logic, mhm. It would be nice to at least move the event policy out to a helper, to keep it the same.

Reply:
Make sense. Will extract to variable